node/src/constants/amplitude.rs

```diff
@@ -0,0 +1,65 @@
+use runtime_common::{Balance, UNIT};
+use spacewalk_primitives::{oracle::Key, Asset, CurrencyId, CurrencyId::XCM, VaultCurrencyPair};
+
+pub const AMPLITUDE_PARACHAIN_ID: u32 = 2124;
+pub const FOUCOCO_PARACHAIN_ID: u32 = 2124;
+pub const AMPLITUDE_INITIAL_ISSUANCE: Balance = 200_000_000 * UNIT;
+pub const INITIAL_ISSUANCE_PER_SIGNATORY: Balance = 200 * UNIT;
+pub const INITIAL_COLLATOR_STAKING: Balance = 10_010 * UNIT;
```

Comment:
We should define this differently here, similar to how it is done [now](https://github.com/pendulum-chain/pendulum/blob/c113f89cb79144ba0a299d5fb211aaac00470227/node/src/constants/pendulum.rs#L8-L9) in Pendulum.

Reasoning: This value is used twice – once to determine how much initial balance collators receive and once to define how much they stake initially. They should not stake everything they receive, though, in order to have some free balance left to pay for transaction fees.

That means: they should receive 10,010 units but only stake 10,000.

Reply:
Great! I created the `COLLATOR_ADDITIONAL` constant in the amplitude constants file, and replaced the value of `INITIAL_COLLATOR_STAKING` from 10_010 to 10_000.

I used `INITIAL_COLLATOR_STAKING` + `COLLATOR_ADDITIONAL` [here](https://github.com/pendulum-chain/pendulum/blob/a1a2ca78b0e4aab3c0f33b0140f568f051410636/node/src/chain_spec.rs#L476C2-L486C14) and [here](https://github.com/pendulum-chain/pendulum/blob/a1a2ca78b0e4aab3c0f33b0140f568f051410636/node/src/chain_spec.rs#L658C2-L668C14).